parser/parser.y

```diff
@@ -10219,6 +10234,15 @@ AdminStmt:
 			JobIDs: $6.([]int64),
 		}
 	}
+|	"ADMIN" "SHOW" "DDL" "JOB" "QUERIES" SelectStmtLimitSimple
+	{
+		ret := &ast.AdminStmt{
+			Tp: ast.AdminShowDDLJobQueriesWithRange,
+		}
+		ret.Limit = $6.(*ast.LimitSimple).Count
+		ret.Offset = $6.(*ast.LimitSimple).Offset
+		$$ = ret
+	}
```

Comment:
We don't need a new stmt type, if we interpret `Limit == 0 && Offset == 0` as the old `AdminShowDDLJobQueries`. What do you think?

Reply:
Maybe it is not very convenient. 
ast.AdminShowDDLJobQueries will be restored to "ADMIN SHOW DDL JOB QUERIES JOB_ID". ast.AdminShowDDLJobQueriesWithRange will be restored to "ADMIN SHOW DDL JOB QUERIES LIMIT n, m", and when limit or offset equals to 0, it would still be printed.  However, we do not want "LIMIT 0, 0" in the restored result of ast.AdminShowDDLJobQueries. If we use the same stmt type and limit and offset are both 0, we cannot judge whether the restored result should contain "LIMIT 0, 0".